python/setup.py.in

```diff
@@ -28,9 +28,10 @@ paddle_bin_dir = 'opt/paddle/bin'
 paddle_bins = ['${PADDLE_BINARY_DIR}/paddle/scripts/paddle_usage',
                '${PADDLE_BINARY_DIR}/paddle/trainer/paddle_trainer',
                '${PADDLE_BINARY_DIR}/paddle/trainer/paddle_merge_model',
-               '${PADDLE_BINARY_DIR}/paddle/pserver/paddle_pserver_main']
+               '${PADDLE_BINARY_DIR}/paddle/pserver/paddle_pserver_main',
+               '${PADDLE_BINARY_DIR}/paddle/scripts/paddle']
 
-paddle_rt_lib_dir = 'local/lib'
+paddle_rt_lib_dir = 'lib'
```

Comment:
Maybe this change would impact "pip install " without docker for user. 
Since the unit test for this has not ready yet @luotao1 

Anyway, if  `/usr/lib` or `/usr/local/lib` are both in `LD_LIBRARY_PATH`, then it won't be a problem for users.